sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/src/Internal/ValidationOptions.cs

```diff
@@ -13,127 +15,83 @@ namespace Microsoft.Azure.WebPubSub.AspNetCore
     /// </summary>
     internal class ValidationOptions
     {
-        private const string EndpointPropertyName = "Endpoint";
-        private const string AccessKeyPropertyName = "AccessKey";
-        private const string PortPropertyName = "Port";
-        private static readonly char[] KeyValueSeparator = { '=' };
-        private static readonly char[] PropertySeparator = { ';' };
         private readonly Dictionary<string, string> _hostKeyMappings = new(StringComparer.OrdinalIgnoreCase);
 
         /// <summary>
-        /// Init the ValidationOptions based on a set of allowed connection strings.
+        /// Create a default ValidationOptions.
         /// </summary>
-        /// <param name="connectionStrings">The upstream connection strings for validation.</param>
-        public ValidationOptions(params string[] connectionStrings)
-        {
-            foreach (var item in connectionStrings)
-            {
-                if (string.IsNullOrEmpty(item))
-                {
-                    continue;
-                }
-                (Uri host, string accessKey) = ParseConnectionString(item);
-                _hostKeyMappings.Add(host.Host, accessKey);
-            }
-        }
+        public ValidationOptions()
+        { }
 
         /// <summary>
-        /// Init the ValidationOptions based on a set of allowed connection strings.
+        /// Init the ValidatoinOptions based on ServiceEndpoint Endpoint and AccessKey.
         /// </summary>
-        /// <param name="connectionStrings">The upstream connection strings for validation.</param>
-        public ValidationOptions(IEnumerable<string> connectionStrings)
-            : this(connectionStrings.ToArray())
+        /// <param name="endpoint"></param>
+        public ValidationOptions(ServiceEndpoint endpoint)
```

Comment:
looks like you support `params ServiceEndpoint[]`?

Reply:
Yes, in future when multiple endpoints are open. Currently only single endpoint is able to add.